apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -1078,11 +1103,16 @@ def _create_shadow_ext_net_for_nat(self, context, l3out_name, ext_epg_name,
                     owner=router_tenant, transaction=trs)
 
                 # link up shadow external-EPG to NAT EPG
-                self.apic_manager.associate_external_epg_to_nat_epg(
-                    router_tenant, shadow_l3out, shadow_ext_epg,
-                    nat_epg_name, target_owner=nat_epg_tenant,
-                    app_profile_name=self._get_network_app_profile(network),
-                    transaction=trs)
+
+                # ASR is the only router we support now so any other values
```

Comment:
still applied. The comment here says that the if statement below means there is no router_type specified at all.